Tetris_formal.py

```diff
@@ -156,159 +169,122 @@ def get_direction(self, color):
         ]
 
         return up, down, right, left
-        
-        
+
     def get_color(self):
-        num = random.randint(1,2)
-        
-        if num == 1 :
-            color = (20,90,90)
-        
-        elif num == 2 :
-            
-            color = pygame.image.load('/Users/coolguy/Documents/租房照片二.jpg')
-            color = pygame.transform.scale(color,(40,40))
-            color.blit(color,(self.x, self.y))
-            
-       
+        """隨機選擇要顏色還是圖片"""
+
+        color = self.color_or_picture(
+            (20, 90, 90), "/Users/coolguy/Documents/租房照片二.jpg"
```

Comment:
不能用你電腦上的路徑，要改用相對路徑，同時上傳照片到github上
```suggestion
            (20, 90, 90), "<Should be relative filepath.>"
```